dockerclient/client.go

```diff
@@ -1093,6 +1106,13 @@ func (e *ClientExecutor) CopyContainer(container *docker.Container, excludes []s
 				}
 				r = filtered
 			}
+			if c.Chmod != "" {
```

Comment:
Should this only happen if c.Chown != "" {?

Reply:
Oof, no, it should not.

Reply:
Wait, it doesn't...?

Reply:
Your right the way github is displaying it confused me.

LGTM